roles/kubernetes/master/templates/kubeadm-config.v1alpha2.yaml.j2

```diff
@@ -159,6 +186,18 @@ schedulerExtraArgs:
   {{ key }}: "{{ kube_kubeadm_scheduler_extra_args[key] }}"
 {% endfor %}
 {% endif %}
+{% if cloud_provider is defined and cloud_provider in ["openstack", "azure", "vsphere"] %}
+  cloud-provider: {{cloud_provider}}
+  cloud-config: {{ kube_config_dir }}/cloud_config
+{% elif cloud_provider is defined and cloud_provider in ["aws", "external"] %}
+  cloud-provider: {{cloud_provider}}
+{% endif %}
+{% if (cloud_provider is defined and cloud_provider in ["openstack", "azure", "vsphere"]) or volume_cross_zone_attachment %}
+schedulerExtraVolumes:
+- name: cloud-config
+  hostPath: {{ kube_config_dir }}/cloud_config
```

Comment:
delete it ,Cloud provider need to configure apiserver、controllerManager and kubelet. not kube-scheduler.